dbt/adapters/snowflake/impl.py

```diff
@@ -246,17 +254,3 @@ def submit_python_job(self, parsed_model: dict, compiled_code: str):
 
     def valid_incremental_strategies(self):
         return ["append", "merge", "delete+insert"]
-
-    @classmethod
-    def render_column_constraint(cls, constraint: ColumnLevelConstraint) -> str:
-        if constraint.type == ConstraintType.check:
```

Comment:
Maybe this is over-engineering it, but can we use `CONSTRAINT_SUPPORT`? So this:
```python
if self.CONSTRAINT_SUPPORT[constraint.type] == ConstraintSupport.NON_SUPPORTED:
    return ""
return super().render_column_constraint(constraint)
```
For that matter, can we just put this in `dbt-core`? Then we wouldn't need to override this method at all.

Reply:
I'm not exactly sure what you mean by override – it seems that they're removing these methods entirely right?

Reply:
I don't read so gud

Reply:
Apparently if you stare at code long enough, you forget what red/green and left/right mean.